### PR TITLE
Add virt-api and virt-controller resource utilization to benchmarks

### LINKED ISSUE #3486: add virt-api and virt-controller resource utilization to benchmarks
**Is your feature request related to a problem? Please describe**:
KubeVirt sig-scale tests are adding metrics to measure cpu and memory utilization of virt-api and virt-controller via this PR https://github.com/kubevirt/kubevirt/pull/12116.

The automation to scrape the metrics and plot graphs based on this needs to be  updated to include the new metrics

**Describe the solution you'd like**:
- [ ] Add the new metrics to the list here to scrape the metrics: https://github.com/kubevirt/project-infra/blob/ffdce4b344999d15e94cbc8e017643849a03a544/robots/cmd/perf-report-creator/scrape.sh#L13-L14 
- [ ] Add the new metrics to the list here to plot the metrics: https://github.com/kubevirt/project-infra/blob/main/robots/cmd/perf-report-creator/graph.sh#L10 

**Describe alternatives you've considered**:
NA

**Additional context**:
None

### PR DESCRIPTION
**Pull request by Sreeja1725**
**What this PR does / why we need it**:

Make virt-api and virt-controller resource utilization metrics to benchmarks available in the graphs.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3486
Complements https://github.com/kubevirt/kubevirt/pull/12116

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
Add virt-api and virt-controller resource utilization to benchmarks
```

### CODE REVIEW

File: robots/cmd/perf-report-creator/graph.sh [hunk 1]
```diff
@@ -7,23 +7,23 @@ perf-report-creator weekly-graph \
...
+  --metrics-list=vmiCreationToRunningSecondsP50,vmiCreationToRunningSecondsP95,LIST-virtualmachineinstances-count,LIST-pods-count,LIST-nodes-count,LIST-virtualmachineinstancemigrations-count,LIST-endpoints-count,GET-virtualmachineinstances-count,GET-pods-count,GET-nodes-count,GET-virtualmachineinstancemigrations-count,GET-endpoints-count,CREATE-virtualmachineinstances-count,CREATE-pods-count,CREATE-nodes-count,CREATE-virtualmachineinstancemigrations-count,CREATE-endpoints-count,PATCH-virtualmachineinstances-count,PATCH-pods-count,PATCH-nodes-count,PATCH-virtualmachineinstancemigrations-count,PATCH-endpoints-count,UPDATE-virtualmachineinstances-count,UPDATE-pods-count,UPDATE-nodes-count,UPDATE-virtualmachineinstancemigrations-count,UPDATE-endpoints-count,avgVirtAPIMemoryUsageInMB,avgVirtControllerMemoryUsageInMB,minVirtAPIMemoryUsageInMB,maxVirtAPIMemoryUsageInMB,minVirtControllerMemoryUsageInMB,maxVirtControllerMemoryUsageInMB,avgVirtAPICPUUsage,avgVirtControllerCPUUsage
 #
-## plot sig-performance 1.29 weekly vm
+## plot 100-density-test results
```
Comment (alaypatel07) [inline robots/cmd/perf-report-creator/graph.sh:20]: will the metrics be available to density tests as well? We need to check and if not available drop this

File: robots/cmd/perf-report-creator/graph.sh [hunk 2]
```diff
@@ -7,23 +7,23 @@ perf-report-creator weekly-graph \
...
+  --metrics-list=vmiCreationToRunningSecondsP50,vmiCreationToRunningSecondsP95,LIST-virtualmachineinstances-count,LIST-pods-count,LIST-nodes-count,LIST-virtualmachineinstancemigrations-count,LIST-endpoints-count,GET-virtualmachineinstances-count,GET-pods-count,GET-nodes-count,GET-virtualmachineinstancemigrations-count,GET-endpoints-count,CREATE-virtualmachineinstances-count,CREATE-pods-count,CREATE-nodes-count,CREATE-virtualmachineinstancemigrations-count,CREATE-endpoints-count,PATCH-virtualmachineinstances-count,PATCH-pods-count,PATCH-nodes-count,PATCH-virtualmachineinstancemigrations-count,PATCH-endpoints-count,UPDATE-virtualmachineinstances-count,UPDATE-pods-count,UPDATE-nodes-count,UPDATE-virtualmachineinstancemigrations-count,UPDATE-endpoints-count,avgVirtAPIMemoryUsageInMB,avgVirtControllerMemoryUsageInMB,minVirtAPIMemoryUsageInMB,maxVirtAPIMemoryUsageInMB,minVirtControllerMemoryUsageInMB,maxVirtControllerMemoryUsageInMB,avgVirtAPICPUUsage,avgVirtControllerCPUUsage
```
Comment (alaypatel07) [inline robots/cmd/perf-report-creator/graph.sh:18]: @Sreeja1725 for the graphs, can we make all the three virt-api metrics and virt-controller metrics together
```
avgVirtAPIMemoryUsageInMB,minVirtAPIMemoryUsageInMB,maxVirtAPIMemoryUsageInMB,avgVirtAPICPUUsage,avgVirtControllerMemoryUsageInMB,minVirtControllerMemoryUsageInMB,maxVirtControllerMemoryUsageInMB,avgVirtControllerCPUUsage
```

**Review by alaypatel07 — COMMENTED** [review]
Added small nit, apart from that changes look good to me

File: robots/cmd/perf-report-creator/graph.sh (hunk 2 above)
Comment (Sreeja1725) [reply robots/cmd/perf-report-creator/graph.sh]: Updated

**Pull request merged**